### PR TITLE
[eclipse/xtext#1606] Update Xtext.setup for Eclipse 2020-03

### PR DESCRIPTION
**Pull request by kthoms**
Also moved 'Oxygen' repository list before 'Photon'.

DO NOT MERGE UNTIL AVAILABILITY OF
- https://download.eclipse.org/tools/orbit/downloads/2020-03
- https://download.eclipse.org/eclipse/updates/4.15-I-builds

### CODE REVIEW

File: releng/org.eclipse.xtext.contributor/Xtext.setup [hunk 1]
```diff
@@ -105,7 +105,7 @@
   <setupTask
       xsi:type="setup:VariableTask"
       name="p2.orbit"
-      value="https://download.eclipse.org/tools/orbit/downloads/2019-12"/>
+      value="https://download.eclipse.org/tools/orbit/downloads/2020-03"/>
```
Comment (cdietrich) [inline releng/org.eclipse.xtext.contributor/Xtext.setup]: is not there yet correct?

Comment (kthoms) [reply releng/org.eclipse.xtext.contributor/Xtext.setup]: As mentioned in the comment above ;)

File: releng/org.eclipse.xtext.contributor/Xtext.setup [hunk 2]
```diff
@@ -1079,6 +1110,8 @@
               url="${p2.emf}"/>
           <repository
               url="${p2.draw2d}"/>
+          <repository
```
Comment (cdietrich) [inline releng/org.eclipse.xtext.contributor/Xtext.setup]: is not there yet, correct?

also shouldnt there be a new section for 2019-12?

**Comment by kthoms** [comment]
2019-12 was already there

screenshot 360

File: releng/org.eclipse.xtext.contributor/Xtext.setup (hunk 2 above)
Comment (cdietrich) [reply releng/org.eclipse.xtext.contributor/Xtext.setup]: ahh the move covers this

**Comment by cdietrich** [comment]
@kthoms the 2020-03 url is there now. so can you test and merge if successful?

**Comment by cdietrich** [comment]
actually the new orbit is still missing

**Comment by cdietrich** [comment]
@kthoms can you please split up this PR into two (one without orbit) so that we can get oomph running again

**Pull request merged**